app/controllers/customers_controller.rb

```diff
@@ -0,0 +1,12 @@
+class CustomersController < ApplicationController
+  def show
+    @customers = Customer.find(params[:id])
+  end
+
+  def create
+    @customer = Customer.find(params[:id])
+    item = Item.find(params[:item_id])
+    @customer.add_item(item.id)
```

Comment:
hmmm, your variable `item` is holding the actual Item object, but then you're only passing the item.id into your `add_item` method, but then in the add_item method, you're doing another db query to find the item associated with this id. Instead, you could just pass this item object into the add_item method.

Reply:
Got it! Just reread your note and this must be the line that got left in when you moved the method to the model.